src/olympia/files/utils.py

```diff
@@ -151,6 +151,9 @@ class RDFExtractor(object):
     TYPES = {'2': amo.ADDON_EXTENSION, '4': amo.ADDON_THEME,
              '8': amo.ADDON_LPAPP, '64': amo.ADDON_DICT,
              EXPERIMENT_TYPE: amo.ADDON_EXTENSION}
+    # Langpacks and dictionaries, if the type is properly set, are always
+    # considered restartless.
+    ALWAYS_RESTARTLESS_TYPES = ('64', '8')
```

Comment:
for tired eyes, could we do something like

```python
ALWAYS_RESTARTLESS_TYPES = (TYPES[amo.ADDON_DICT], TYPES[amo.ADDON_LAPP])
```

although it's not much more readable… maybe nevermind :laughing: